createrelease.bat

```diff
@@ -7,6 +7,7 @@ copy "OWML.Patcher\bin\Debug\OWML.Patcher.dll" "Release\OWML.Patcher.dll"
 copy "OWML.ModLoader\bin\Debug\OWML.ModLoader.dll" "Release\OWML.ModLoader.dll"
 copy "OWML.Launcher\bin\Debug\OWML.Launcher.exe" "Release\OWML.Launcher.exe"
 copy "OWML.Launcher\bin\Debug\OWML.Config.json" "Release\OWML.Config.json"
+copy "OWML.Launcher\bin\Debug\OWML.DefaultConfig.json" "Release\OWML.DefaultConfig.json"
```

Comment:
Now that we have a default OWML config, we shouldn't include OWML.Config.json in the releases, right? One of the main advantages of having a separate default config file is that new releases don't replace configs of previous releases (which can be very annoying if you need a custom game path). OMLW.Config.json should only be generated from the default, after OWML starts.

Reply:
Yeah, totally 👍 
If we do this: before starting OWML the first time, how will you show/edit the config in MM?

Reply:
Same as for mods. If mod manager sees only default, copy default to user config. Otherwise edit user config.

Need to make sure we copy each property and not only the whole file (in case we add new properties to config). Don't remember if mods already do this.

Reply:
@Raicuparta removed config file, and creating it from default if it's missing.

Reply:
Just copying the file, as there's no (reason to have a) list of generic settings.